apps/mull-ui/src/app/pages/messages/event-chat/event-chat.tsx

```diff
@@ -64,28 +71,52 @@ export const EventChat = ({ eventId, channelName, restrictChatInput }: EventChat
       },
     });
 
+  /**
+   * Handles image file uploads
+   * @param {ChangeEvent<HTMLInputElement>} event
+   */
+  const handleFileUpload = (event: ChangeEvent<HTMLInputElement>) => {
+    setImageURLFile(URL.createObjectURL(event.target.files[0]));
+    setFile(event.target.files[0]);
+    formik.setFieldValue('imageFile', event.target.files[0]);
+  };
```

Comment:
💡Since image file uploads are being done in multiple places now, would it be possible to abstract out the common parts to a helper function or something?

Reply:
Yes. but not in this ticket